zerver/lib/hotspots.py

```diff
@@ -89,6 +89,9 @@ def to_dict(self) -> Dict[str, str]:
     OneTimeNotice(
         name="intro_recent_view_modal",
     ),
+    OneTimeNotice(
+        name="messages_fading_banner",
```

Comment:
Would `intro_message_fading` be a better name? I realize that we used the `_banner` / `_modal` suffice on previous ones, but I'm thinking that if we were to, in the future, change this from a banner to a modal, we probably wouldn't want to have to rename the field.

Reply:
Yeah makes sense from future perspective. Renamed it to `intro_messages_fading`